https://bugs.gentoo.org/show_bug.cgi?id=537684
Patch by Michael Weiser

Stripped down to c-file part only to avoid autoreconf triggering.

--- ./toolchain-prefix-wrapper-0.3.1723/ld/darwinplugin.c.orig	2015-01-24 21:23:44.000000000 +0100
+++ ./toolchain-prefix-wrapper-0.3.1723/ld/darwinplugin.c	2015-01-24 21:24:04.000000000 +0100
@@ -22,8 +22,6 @@
 		err = -1;
 
 		if (StringListAppendConcat(data->out->argList, "-search_paths_first", 19, NULL) < 0) break;
-		if (StringListAppendConcat(data->out->argList, "-macosx_version_min", 19, NULL) < 0) break;
-		if (StringListAppendConcat(data->out->argList, MIN_OSX_TARGET, sizeof(MIN_OSX_TARGET), NULL) < 0) break;
 
 
 		/* keep argv[1] ... argv[n] */
